sdk/storage/azure-storage-common/CMakeLists.txt

```diff
@@ -52,6 +52,7 @@ set(
     inc/azure/storage/common/internal/file_io.hpp
     inc/azure/storage/common/internal/reliable_stream.hpp
     inc/azure/storage/common/internal/shared_key_policy.hpp
+    inc/azure/storage/common/internal/shared_key_policy_lite.hpp
```

Comment:
I understand there's debate about where this file should go. Can you share more details? Like the pros and cons of putting this file in storage-common package vs tables package.

Reply:
We'll have a meeting in January probably, when Larry gets back regarding the dependency on storage common , and if this should be here, in azure common or somwhere else.